### ISSUE TITLE
Update or remove image alt text

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/unitywebagency#14]
- [x] Team portraits contain the image@2x as their alt text
- [x] Remove alt text on decorative team images (mushrooms, roller skates, etc). This is encountered before the bio text in the DOM and is a little awkward / without context as a result.
- [x] Partner logos should have their alt text removed in this context... the image alt text containing the partner name is read aloud following by the same exact text in the following link.
